packages/client/src/components/gamePause.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC } from 'react'
+import Modal from './modal'
+import Button from '../ui/button'
+import styled from 'styled-components'
+import IconSettings from '../assets/icons/IconSettings'
+import IconStar from '../assets/icons/IconStar'
+import { useNavigate } from 'react-router'
+import IconForum from '../assets/icons/IconForum'
+
+type Props = {
+  visible: boolean
+  handleClose: () => void
+}
+const GamePause: FC<Props> = (props) => {
+  const navigate = useNavigate()
+
+  const handleClick = (path?: string) => () => {
+    if (path) navigate(path)
```

Comment:
Если очень хочется однастрочная запись, то можно так:
```suggestion
    path && navigate(path)
```